frontend/www/js/omegaup/problem/details.ts

```diff
@@ -179,10 +178,6 @@ OmegaUp.on('ready', () => {
               })
               .catch(ui.apiError);
           },
-          'tab-selected': (tabName: string) => {
-            arenaInstance.activeTab = tabName;
-            window.location.replace(`#${arenaInstance.activeTab}`);
-          },
```

Comment:
esto se puede quedar casi como estaba:

```suggestion
          'update:activeTab': (tabName: string) => {
            window.location.replace(`#${arenaInstance.activeTab}`);
          },
```

Reply:
Funcionó!! Gracias